src/main/java/commands/DeleteCommand.java

```diff
@@ -0,0 +1,19 @@
+package commands;
+
+import core.Ui;
```

Comment:
I like how imports are arranged alphabetically from well-named packages.

Reply:
For readability!